patches/uClibc-ng/1.0.12/100-provide-_obstack_free.patch

```diff
@@ -0,0 +1,12 @@
+diff -ur uClibc-ng-1.0.12.orig/libc/misc/gnu/obstack.c uClibc-ng-1.0.12/libc/misc/gnu/obstack.c
+--- uClibc-ng-1.0.12.orig/libc/misc/gnu/obstack.c	2016-03-13 15:08:44.408962824 -0700
++++ uClibc-ng-1.0.12/libc/misc/gnu/obstack.c	2016-03-13 15:13:30.129322998 -0700
+@@ -385,7 +385,7 @@
+     abort ();
+ }
+ 
+-# if 0
++# if 1
+ /* Older versions of libc used a function _obstack_free intended to be
+    called by non-GCC compilers.  */
+ strong_alias (obstack_free, _obstack_free)
```

Comment:
Hi,

can you sent a patch to our mailing list with signed-off-by? 
I have disabled obstack in my case, because I see no application using it.
In the git history it was added for reiserfsprogs, which is rather dead I think.
But I am happy to fix it for the next release.